lib/beacon_web/components/layouts.ex

```diff
@@ -70,24 +70,68 @@ defmodule BeaconWeb.Layouts do
     ""
   end
 
-  def page_meta_tags(%{__dynamic_page_id__: _, __site__: _} = assigns) do
-    {:safe, page_meta_tags_unsafe(assigns)}
+  def merge_meta_tags(assigns) do
+    site_meta_tags = site_get_meta_tags()
+
+    layout_meta_tags = layout_get_meta_tags(assigns)
+
+    page_meta_tags = page_get_meta_tags(assigns)
+
+    page_meta_tags
+    |> merge_tags(layout_meta_tags)
+    |> filter_out_attr_value("csrf-token")
+    |> merge_tags(site_meta_tags)
```

Comment:
Concatenation does the job here:

```suggestion
    site_meta_tags = site_get_meta_tags()
    layout_meta_tags = layout_get_meta_tags(assigns)
    page_meta_tags = page_get_meta_tags(assigns)

    page_meta_tags = Enum.reject(page_meta_tags, & &1["name"] == "csrf-token")
    site_meta_tags ++ layout_meta_tags ++ page_meta_tags
```

The bottom line is to allow duplicate meta tags because that's indeed allowed, for eg `og` properties and also regular meta tags like the `http-equiv` directive. And leverage the list order.